Docs/radium-as-submodule.md

```diff
@@ -88,3 +87,5 @@ mkdir build-radium-release
 cmake ../Radium-Engine -DCMAKE_BUILD_TYPE=Release
 make -j8
 ```
+Or add Radium to your project cmake process with `ExternalProject_Add`.
+
```

Comment:
@hoshiryu Do you have a valid example for this one ?
I prefer to compile radium on its own, as it happens to delete my build dir, but don't want to recompile radium. Both approaches are ok.

Reply:
Well, it's the same process as the one we use for Radium's external libraries.
If your project uses Radium only as an external lib, you only need to compile Radium once.
Maybe a Radium "install" might be needed, as we do for the external libs.

Reply:
could you add the valid example command to the doc ?
with external project in radium, assimp for instance, if I clean radium build dir, then next compilation will compile assimp right ?

Reply:
If you use make clean to clean your build dir it's fine, but if you `rm -rf build` you would have to rebuild it again.
A way to do it might be to check if the libs are already compile and call the ExternalProject_add command only if they are not.